django/utils/log.py

```diff
@@ -27,12 +28,23 @@
             '()': 'django.utils.log.RequireDebugTrue',
         },
     },
+    'formatters': {
+        'django.request.runserver': {
+            '()': 'django.utils.log.RunserverFormatter'
+        }
+    },
     'handlers': {
         'console': {
             'level': 'INFO',
             'filters': ['require_debug_true'],
             'class': 'logging.StreamHandler',
         },
+        'django.request.runserver': {
+            'level': 'INFO',
+            'filters': ['require_debug_true'],
```

Comment:
technically, this is a change in behaviour. Currently on `master` the `DEBUG` setting has no effect on whatever the requests to `runserver` are printed out – ie: they are always printed out.

The filter seems appropriate, but I can remove if you'd rather.